src/org/exist/client/InteractiveClient.java

```diff
@@ -1309,18 +1346,15 @@ private synchronized boolean findRecursive(final Collection collection, final Pa
     /**
      * stores given Resource
      *
-     * @param fileName simple file or directory
+     * @param file file or directory
      * @throws XMLDBException
      */
-    protected synchronized boolean parse(String fileName) throws XMLDBException {
+    protected synchronized boolean parse(final Path file) throws XMLDBException {
```

Comment:
@wolfgangmm we need you because `fileName.replace('/', java.io.File.separatorChar).replace('\\', java.io.File.separatorChar);` here from [1st commit](https://github.com/adamretter/exist/commit/7ac94759168fc331419ad4d5777c5c07187d208e#diff-e50a0f4a572caabd4804d0cd924aa666R1088).

Will you able to recall why it was done this way? (with hope)